src/analysis/souffle/authorization_logic.dl

```diff
@@ -52,7 +54,29 @@ isPrincipal(speaker) :- saysDowngrades(speaker, _, _).
 //-----------------------------------------------------------------------------
 downgrades(path, tag) :- ownsTag(owner, tag), saysDowngrades(owner, path, tag).
 
+claimNotHasTag(path, tag) :- downgrades(path, tag).
```

Comment:
What does this additional indirection buy us? Why not use `downgrades` directly as we used to?

Reply:
I think it doesn't actually buy us much and we can get rid of it. As of this PR, I was thinking that you and Mark wanted more encapsulation between auth logic and DFA, and perhaps having a relation that looks like `claim path is NOT tag` would help with this encapsulation somehow?

Probably we should get rid of this extra indirection and keep it as downgrades.